### PR TITLE
Add a method to display the UID from the console

### LINKED ISSUE #186: Add a console command to print the UID

### PR DESCRIPTION
**Pull request by nomis52**
Closes #186

### CODE REVIEW

File: common/uid_store.h [hunk 1]
```diff
@@ -42,6 +42,13 @@ const uint8_t* UIDStore_GetUID();
  * @param output The location to store the string in, must be at least
  *   UID_LENGTH * 2 + 1 characters.
  */
+void UIDStore_AsAsciiString(char *output);
+
+/**
+ * @brief Convert the UID to a string representation, e.g. abcd:01020304
```
Comment (peternewman) [inline common/uid_store.h]: Can you differentiate these somehow in the Doxygen @brief please.

**Comment by peternewman** [comment]
Generally LGTM

File: common/uid_store.h (hunk 1 above)
Comment (nomis52) [inline common/uid_store.h]: Done

**Comment by peternewman** [comment]
LGTM

**Pull request merged**